internal/bminventory/inventory.go

```diff
@@ -1965,10 +1951,27 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 		}
 	}
 
+	if params.ClusterUpdateParams.MachineNetworkCidr != nil && common.IsSingleNodeCluster(cluster) {
+		err = network.VerifyMachineCIDR(swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr), cluster.Hosts, log)
+		if err != nil {
+			return err
+		}
+		err = setCommonUserNetworkManagedParams(params.ClusterUpdateParams, cluster, updates, log)
+		if err != nil {
+			return err
+		}
+		machineCidr = swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr)
+		setMachineNetworkCIDRForUpdate(updates, machineCidr)
+	}
+
 	if err = network.VerifyClusterCIDRsNotOverlap(machineCidr, clusterCidr, serviceCidr, userManagedNetworking); err != nil {
 		return common.NewApiError(http.StatusBadRequest, err)
 	}
 
+	if err = updateNtpSources(params, updates, log); err != nil {
```

Comment:
Where did this come from?

Reply:
just moved it cause this function had too many "ifs" and failed the linter